# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: 228200dcaf93
# Full Hash: 228200dcaf93bb3f10e338157325ef6227e8a0a5
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-02-15 18:16:35
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r b25110652394 -r 228200dcaf93 modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Wed Feb 15 08:29:41 2023 +0000
+++ b/modules/libpref/init/all.js	Wed Feb 15 08:29:41 2023 +0000
@@ -454,8 +454,6 @@
   pref("gfx.hidpi.enabled", 2);
 #endif
 
-pref("gfx.color_management.display_profile", "");
-
 pref("gfx.downloadable_fonts.enabled", true);
 pref("gfx.downloadable_fonts.fallback_delay", 3000);
 pref("gfx.downloadable_fonts.fallback_delay_short", 100);
